doc/website/advanced/usage-guide.md

```diff
@@ -79,3 +97,28 @@ count = 100
 ```
 
 When no config file is specified, a hard-coded version similar to [default config](../../../iceoryx_posh/etc/iceoryx/roudi_config_example.toml) will be used.
+
+### static configuration
+
+Another way is to have a static config which is compile-time dependent, this means that you have to recompile your RouDi application if you want to change your config (not the iceoryx_posh_roudi lib).
+You can have your own sourcefile with `main()` method where you can create your custom configuration and pass it to a RouDi instantiation. 
+In your CMake file for you custom RouDi you need to ensure that it is **not** linking against `iceoryx_posh_config` to have a static config.
+
+A good example how a static config could look like can be found [here](../../../iceoryx_examples/iceperf/roudi_main_static_config.cpp).
+
+## Iceoryx library build
```

Comment:
Doesn't this section fit better to `installation-guide-for-contributors.md`? No problem to do this later